bp_me/src/asm/include/bp_common_me_if.h

```diff
@@ -95,12 +95,12 @@ typedef enum {
 
 // Mem Commands
 typedef enum {
-  e_cce_mem_rd               = 0
-  ,e_cce_mem_wr              = 1
-  ,e_cce_mem_uc_rd           = 2
-  ,e_cce_mem_uc_wr           = 3
-  ,e_cce_mem_pre             = 4
-} bp_cce_mem_cmd_type_e;
+  e_bp_mem_rd               = 0
```

Comment:
@muwyse what do you think of the names here? e_mem_rd is probably too likely to cause name conflicts.  But we usually don't use _bp_ in enums

Reply:
One option that would align with the bp_mem_msg_size_e enum would be to call the enum type bp_mem_msg_e and each name would be e_mem_msg_XXX (e_mem_msg_rd, e_mem_msg_wr, etc.)

A slightly more verbose option is bp_mem_msg_type_e and e_mem_msg_type_XXX

Reply:
agreed that e_mem_rd is a likely name conflict waiting to happen

Reply:
I vote for the bp_mem_msg_type_e and e_mem_msg_type_XXX, because bp_mem_msg_e is similar to bp_``name_mp``_msg_s, the top level message struct.

Reply:
bp_mem_msg_type_e and e_mem_msg_XXX sounds good!